examples/tutorial/jupyter/README.md

```diff
@@ -15,8 +15,10 @@ please install every package listed in `requirements.txt` file individually with
 
 ## Run Jupyter Notebooks
 
-A Jupyter Notebook example can be run from the current directory as follows:
+A Jupyter Notebook server can be run from the current directory as follows:
 
 ```bash
-jupyter notebook local/exercise_1.ipynb
+jupyter notebook
 ```
+
+Navigate to a concrete notebook (for example to the `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb`).
```

Comment:
```suggestion
Navigate to a concrete notebook (for example, to the `execution/pandas_on_ray/local/exercise_1.ipynb`).
```
Since a jupyter server is run from the current directory we should use relative paths to notebooks.